src/components/common/Job/saveJobButton.tsx

```diff
@@ -0,0 +1,42 @@
+import * as React from 'react';
+
+import { WithStyles, withStyles } from '@material-ui/core/styles';
+import StarBorderOutlinedIcon from '@material-ui/icons/StarBorderOutlined';
+import FullStarIcon from '@material-ui/icons/Star'
+import Button from '@material-ui/core/Button';
+
+import styles from './jobStyles';
+
+interface SaveJobData {
+    isFavorite: boolean;
+    handleClick: () => void;
+}
+
+type SaveJobProps = SaveJobData & WithStyles<typeof styles>;
+
+const SaveJobButton: React.FC<SaveJobProps> = (props): JSX.Element => {
+    const { classes, isFavorite, handleClick } = props;
+
+    const renderStarIcon = (): JSX.Element => {
+        return isFavorite
+            ? <FullStarIcon className={classes.btnIcon} />
+            : <StarBorderOutlinedIcon className={classes.btnIcon} />
+    }
+
+    return (
+        <Button 
+            className={`${classes.jobBtn} ${isFavorite && classes.boldText}`}
+            onClick={handleClick}
+            startIcon={renderStarIcon()}
+        >
+            {isFavorite
```

Comment:
If the trinary fits a single line, it's prefered to keep it like that

Reply:
I agree, but i doesn't fit in a single line in this case